extensions/whentomeet/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "whentomeet",
+  "title": "WhenToMeet",
+  "description": "Create WhenToMeet scheduling events using natural language. Describe your event and get a pre-filled WhenToMeet URL instantly.",
+  "icon": "extension_icon.png",
+  "author": "felix_wortmann",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "version": "2.0.0",
+  "keywords": [
+    "whentomeet",
+    "scheduling",
+    "calendar",
+    "meeting",
+    "event",
+    "productivity"
+  ],
+  "commands": [
+    {
+      "name": "create-event",
+      "title": "Create Whentomeet Event Using AI",
+      "subtitle": "WhenToMeet",
```

Comment:
We can remove the subtitle as the title contains the name as context 👍

Reply:
Just removed it